### PR TITLE
Updating How Bad Standing Votes Work

### PR DESCRIPTION
**Pull request by saucetray**
Check one:
- [x] Semantic Change: something about the meaning of the text is different
- [ ] Non-semantic Change: Spelling, grammar, or formatting changes.

Summary of change(s):
The way this is written is so bad. Theoretically, people voted to good standing should now be active and given off floor status which is just... bad.

### CODE REVIEW

**Comment by mxmeinhold** [comment]
Looks good.
One question, why remove the requirement to announce it?

**Comment by saucetray** [comment]
Is it ever announced now?

**Comment by mxmeinhold** [comment]
Hmm. I don’t feel a need to announce it so long as it’s included in notes I suppose.

File: articles.tex [hunk 1]
```diff
@@ -184,8 +184,7 @@
 Qualifying members may self-select into Active Membership by paying dues to the Financial Director and notifying the Evaluations Director.
 \\*\\*
 Any Alumni Member in bad standing (\ref{Alumni Membership Selection}) may become an Active Member by notifying the Evaluations Director of their intent to participate, who will bring them up for vote at the next Evaluations meeting.
```
Comment (csssuf) [inline articles.tex]: This line should be dropped as well.

File: articles.tex [hunk 2]
```diff
@@ -184,8 +184,7 @@
...
-If a majority is in favor of the return, the Alumni is reinstated as an Active Member with Off-Floor status effective immediately.
-The decision will be announced at the following House meeting.
+If a majority is in favor of the return, the Alumni is removed from bad standing.
```
Comment (csssuf) [inline articles.tex]:
```suggestion
If a majority is in favor of the return, the Alumni is considered to be in good standing.
```

**Pull request closed without merging**